clients/lsp-camel.el

```diff
@@ -0,0 +1,68 @@
+;;; lsp-camel.el --- LSP Camel server integration        -*- lexical-binding: t; -*-
+
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-camel nil
+ "LSP support for Camel, using camel-language-server"
+ :group 'lsp-mode
+ :tag "Language Server"
+ :package-version '(lsp-mode . "8.0.0"))
+
+;; Define a variable to store camel language server jar version
+(defconst lsp-camel-jar-version "1.5.0")
+
+;; Define a variable to store camel language server jar name
+(defconst lsp-camel-jar-name (format "camel-lsp-server-%s.jar" lsp-camel-jar-version))
+
+;; Directory in which the servers will be installed. Lsp Server Install Dir: ~/.emacs.d/.cache/camells
+(defcustom lsp-camel-jar-file (f-join lsp-server-install-dir "camells" lsp-camel-jar-name)
+ "Camel Language server jar command."
+ :type 'string
+ :group 'lsp-camel
+ :type 'file
+ :package-version '(lsp-mode . "8.0.0"))
+
+(defcustom lsp-camel-jar-download-url
+ (format "https://repo1.maven.org/maven2/com/github/camel-tooling/camel-lsp-server/%s/%s" lsp-camel-jar-version lsp-camel-jar-name)
+ "Automatic download url for lsp-camel."
+ :type 'string
+ :group 'lsp-camel
+ :package-version '(lsp-mode . "8.0.0"))
```

Comment:
can you change all of these to 8.0.1. Also, can you format the buffer via `C-x h M-x indent-region`?

Reply:
Hello @yyoncho  
Done with the changes, can you please review it.